### PR TITLE
[Doc] Added `JAVA_HOME` details to breaking changes

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: docs/static/breaking-changes.asciidoc [hunk 1]
```diff
@@ -54,6 +54,14 @@ inputs that are ambiguous or illegal. Configurability is removed in 8.0. Now
 {ls} rejects ambiguous and illegal inputs as standard behavior.
 // end::notable-breaking-changes[]
 
+[discrete]
+[[bc-java-home]]
+===== Removed support for JAVA_HOME
+Support for using `JAVA_HOME` to override the path to the JDK that Logstash runs with has been removed for this release.
+In the `8.x` release, users should set the value of `LS_JAVA_HOME` to their preferred JDK path if they
```
Comment (karenzone) [inline docs/static/breaking-changes.asciidoc:61]:
```suggestion
In the `8.x` release, users should set the value of `LS_JAVA_HOME` to the path of their preferred JDK if they
```

File: docs/static/breaking-changes.asciidoc [hunk 2]
```diff
@@ -54,6 +54,14 @@ inputs that are ambiguous or illegal. Configurability is removed in 8.0. Now
...
+wish to use something other than the bundled JDK. The value of `JAVA_HOME` will be ignored.
```
Comment (karenzone) [inline docs/static/breaking-changes.asciidoc:62]:
```suggestion
wish to use a version other than the bundled JDK. The value of `JAVA_HOME` will be ignored.
```

**Review by karenzone — APPROVED** [review]
Minor comments inline for your consideration. Otherwise, LGTM

**Comment by robbavey** [comment]
Committed in #13228

**Pull request closed without merging**